reference/catalog/categories_catalog.v3.yml

```
@@ -1783,8 +1783,9 @@ paths:
           content:
             application/json:
               schema:
-                type: object
-                properties: {}
+                type: array
+                items:
+                  type: object
```

what are the expected properties here?

```suggestion
                  type: object
                  properties: {}
```

---

@slsriehl I added the properties.